internal/events/aggregator_test.go

```diff
@@ -936,12 +937,11 @@ func TestAttemptMessageDispatchFailValidateBadSystem(t *testing.T) {
 	ag, cancel := newTestAggregator()
 	defer cancel()
 
-	msh := ag.syshandlers.(*syshandlersmocks.SystemHandlers)
-	msh.On("HandleSystemBroadcast", mock.Anything, mock.Anything, mock.Anything).Return(false, nil)
-
 	mdm := ag.data.(*datamocks.Manager)
 	mdm.On("GetMessageData", ag.ctx, mock.Anything, true).Return([]*fftypes.Data{}, true, nil)
 
+	mdm.On("ValidateAll", ag.ctx, mock.Anything).Return(false, nil)
+
```

Comment:
@peterbroadhurst This is one of the tests I had to change

Reply:
Just to check, why wouldn't you just change `msh := ag.syshandlers.(*syshandlersmocks.SystemHandlers)` to be the DefinitionHandlers, and return false and set `Header.Type` on the message in `attemptMessageDispatch`?

e.g. so you're testing the same path.